### ISSUE TITLE
possibility of training on pdf shots

### ISSUE THREAD

**Issue by nicoosokhan** [MaybeShewill-CV/CRNN_Tensorflow#247]
Hi, Thanks for the repository.

I'm looking for a CRNN model based on multiple line text recognition.
I have seen your package in "2019.4.10 Updates" has a "demo to recognize Chinese pdf".

The question is: Is this just a pre-trained model for a demo or A a new model that could be trained on a custom dataset.

More clear, How can I train my own dataset on your model.
(The dataset is a collection of (image, text) pair where the text is the whole document written in the image.)

**Comment by MaybeShewill-CV** [comment]
@nicoosokhan No new model here I just use the pretrained crnn model to recognize the chinese pdf sequence:)

**Comment by nicoosokhan** [comment]
So It means that this model is useful for multiple sentences (long labels), right?

**Comment by MaybeShewill-CV** [comment]
@nicoosokhan You must extract the sequence area first. For example
![1](https://user-images.githubusercontent.com/15725187/57304265-95049900-7111-11e9-9de8-251f0dca458c.png) Qualified for predicting

![2](https://user-images.githubusercontent.com/15725187/57304296-a483e200-7111-11e9-869c-d78189a6070c.png) Not qualified for predicting

**Comment by nicoosokhan** [comment]
Extracting means: cropping the image using the information received from a "Detection Model", right?

For the final guide. please tell me:

1) How did you extract those areas?

2) What about a unified model. A model that doesn't need extracting single areas and it process the entire image automatically. I have tried to customize the "feature map to sequences" layer to handle at least two lines of words. But no result.
Do you have any idea of handling multiple lines automatically?

Thanks for valuing the question.

**Comment by MaybeShewill-CV** [comment]
@nicoosokhan 1.Yep extracting means: cropping the image using the information received from a "Detection Model". 2. You may find the solution for extracting text area of a pdf in the test script here https://github.com/MaybeShewill-CV/CRNN_Tensorflow/blob/master/tools/recongnize_chinese_pdf.py 3.You need to figure how to generate multiple sequence features for multiple lines according to the network:)
